pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -775,39 +775,37 @@ func parseOneSystemAdapterConfig(getconfigCtx *getconfigContext,
 		switch port.Dhcp {
 		case types.DT_STATIC:
 			if port.AddrSubnet == "" {
-				errStr := fmt.Sprintf("parseSystemAdapterConfig: Port %s DT_STATIC but missing "+
-					"subnet address in %+v; ignored", port.IfName, port)
-				log.Error(errStr)
-				port.ParseError = errStr
-				port.ParseErrorTime = time.Now()
+				errStr := fmt.Sprintf("Port %s Configured as DT_STATIC but "+
+					"missing subnet address. Port: %+v.", port.IfName, port)
+				log.Errorf("parseSystemAdapterConfig: %s", errStr)
+				port.SetErrorNow(errStr)
 			}
 		case types.DT_CLIENT:
 			// Do nothing
 		case types.DT_NONE:
 			if isMgmt {
-				errStr := fmt.Sprintf("parseSystemAdapterConfig: Port %s: isMgmt with DT_NONE not supported",
-					port.IfName)
-				log.Error(errStr)
-				port.ParseError = errStr
-				port.ParseErrorTime = time.Now()
+				errStr := fmt.Sprintf("Port %s configured as Management port "+
+					"with DHCP type DT_NONE. Only Client and Static are "+
+					"DHCP modes for management ports", port.IfName)
```

Comment:
I don't know what DT_NONE is called in the UI.Maybe say
"with an unsupported DHCP type %d. Client and static are the only allowed DHCP modes for management ports".

Reply:
Done.. Changed it to:

	errStr := fmt.Sprintf("Port %s configured as Management port "+
					"with an unsupported DHCP type %d. Client and static are "+
					"the only allowed DHCP modes for management ports.",
					port.IfName, types.DT_NONE)